src/stores/businessStore.ts

```diff
@@ -97,9 +97,9 @@ export const useBusinessStore = defineStore('business', {
         return state.businessInfo.legalName
       }
       if (this.isFirm) {
-        return this.getAlternateName || 'Unknown'
+        return this.getAlternateName
       } else {
-        return state.businessInfo.legalName || 'Unknown'
+        return state.businessInfo.legalName
       }
```

Comment:
So we have a situation here where this sometimes returns null/empty, and it's handled differently throughout the code.

I'm kinda thing that -- in the future -- we should get rid of this store getter and just fetch whatever name we need, and fallback however we need, right in the components that currently use this. It might seem like a step backwards, but it would remove any assumption about how this is used. Thoughts?

Reply:
Ah, you're talking about how the code handles that scenario by having `Unknown Name` right? Yup, makes a lot of sense to me.

OK to deal with that later on Sev?

Reply:
Yes, it's a problem for later.